node/src/command.rs

```diff
@@ -349,7 +349,7 @@ pub fn run_with(cli: Cli) -> Result<()> {
                 runner.sync_run(|config| cmd.run::<Block, DolphinRuntimeExecutor>(config))
             } else {
                 Err("Benchmarking wasn't enabled when building the node. \
-				You can enable it with `--features runtime-benchmarks`."
+                You can enable it with `--features runtime-benchmarks`."
```

Comment:
Also what about these. Is rust automatically discarding leading whitespace in strings?

Reply:
answering myself: yes it does